test/benchmarks.py

```diff
@@ -42,7 +42,15 @@ def benchmark(model, csv_file, input_fn=genImage, iters=100, warmup=10):
         for _ in range(iters):
           _ = trace_tvm(*inputs)
         tvm_time = time.time() - start
-        print("Done benchmarking TVM")
+        with torch.autograd.profiler.profile() as prof:
+          _ = trace_tvm(*inputs)
+        tvm_profiled_time = 0
```

Comment:
hmmm shouldn't this whole block be under the indent of the `with` block? I thought `prof` will only be visible inside the block

Reply:
nope, with doesn't create a scope